extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsole.java

```diff
@@ -174,7 +176,7 @@ public Object apply(String s, Throwable throwable) {
                 if (throwable != null) {
                     event.fail(throwable);
                 } else {
-                    event.response().end(s);
+                    event.response().putHeader(HttpHeaderNames.CONTENT_TYPE, HTML_CONTENT_TYPE).end(s);
```

Comment:
You don't need to set the contentType header again, or?

Reply:
Hm, the `renderTemplate()` method is called from the `handle()` above...

Reply:
Okay, changed to make it a little more clear